src/Tasks/NativeMethods.cs

```diff
@@ -1050,8 +1050,12 @@ internal static bool AllDrivesMapped()
         /// and then again to pass the client-allocated character buffer. StringBuilder is the most straightforward way
         /// to allocate a mutable buffer of characters and pass it around.
         /// </summary>
+        /// <param name="cacheFlags">Value that indicates the source of the cached assembly.</param>
+        /// <param name="cachePath">The returned pointer to the path.</param>
+        /// <param name="pcchPath">The requested maximum length of CachePath, and upon return, the actual length of CachePath.</param>
+        /// 
         [DllImport("fusion.dll", CharSet = CharSet.Unicode)]
-        internal static extern int GetCachePath(AssemblyCacheFlags cacheFlags, StringBuilder cachePath, ref int pcchPath);
+        internal static extern int GetCachePath([In] AssemblyCacheFlags cacheFlags, char[] cachePath, ref int pcchPath);
```

Comment:
nits:
- `[In]` on an enum makes no difference, when passed by value it is always "in".
- `cachePath` should be `[Out]`; it may not make a difference either if the runtime pins the array but it's a good practice to specify it nonetheless.